src/Tests/ResponseTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ProjectManagementApi\Tests;
+
+require '../../vendor/autoload.php';
+
+use PHPUnit\Framework\TestCase;
+use ProjectManagementApi\Response;
+
+class ResponseTest extends TestCase
+{
+    public function testSuccessJsonSerialize()
+    {
+        $expectedOutput = ['message' => 'Successfully retrieved projects', 'data' => [['id' => 17, 'name' => 'Overhold', 'client_id' => 7, 'deadline' => '30/06/2022', 'overdue' => true], ['id' => 18, 'name' => 'Wrapsafe', 'client_id' => 18, 'deadline' => '28/03/2024', 'overdue' => false]]];
+        
+        $message = 'Successfully retrieved projects';
+
+        $data = [['id' => 17, 'name' => 'Overhold', 'client_id' => 7, 'deadline' => '30/06/2022', 'overdue' => true], ['id' => 18, 'name' => 'Wrapsafe', 'client_id' => 18, 'deadline' => '28/03/2024', 'overdue' => false]];
+
+        $response = new Response($message, $data);
+        
+        $actualOutput = $response->jsonSerialize();
+        $this->assertEquals($expectedOutput, $actualOutput);
```

Comment:
i think, for readability, the order in here should be more like we've learned it in class, ie.:
message
data
response
expected
actual
this